Subject: Change the default to alt_entity=0
Author: Olaf Dabrunz <od@suse.de>
Origin: https://build.opensuse.org/package/view_file/openSUSE:Factory/w3m/w3m-0.5.1-no-ASCII-equivalents-by-default.patch?expand=1
Bug-Novell: https://bugzilla.novell.com/show_bug.cgi?id=247397

    change the default for the option "Use ASCII equivalents to
    display entities" from YES to NO.

diff --git a/fm.h b/fm.h
index 149a02f..49af0e4 100644
--- a/fm.h
+++ b/fm.h
@@ -1089,7 +1089,7 @@ global char SimplePreserveSpace init(FALSE);
 #define wc_Str_conv(x,charset0,charset1) (x)
 #define wc_Str_conv_strict(x,charset0,charset1) (x)
 #endif
-global char UseAltEntity init(TRUE);
+global char UseAltEntity init(FALSE);
 #define GRAPHIC_CHAR_ASCII 2
 #define GRAPHIC_CHAR_DEC 1
 #define GRAPHIC_CHAR_CHARSET 0
